eng/scripts/vcpkg.ps1

```diff
@@ -0,0 +1,52 @@
+[CmdletBinding()]
+Param (
+    [Parameter()]
+    [ValidateNotNullOrEmpty()]
+    [string] $Ref = 'master',
```

Comment:
What's the purpose of this?

Reply:
This is the ref to check out. I defaulted it to `master` but now it defaults to reading the vcpkg txt file.